devops/ci/Jenkinsfile.ci.integration

```diff
@@ -55,15 +57,32 @@ def buildUnitDocker() {
     def uid = sh(returnStdout: true, script: 'id -u').trim()
     def gid = sh(returnStdout: true, script: 'id -g').trim()
     return buildDocker(
-        dockerImage,
+        dockerImageUnitName,
         "devops/ci/Dockerfile.unit",
         "api",
-        "--build-arg uid=$uid --build-arg gid=$gid"
+        "--build-arg TEST_USER=${testUser} --build-arg uid=$uid --build-arg gid=$gid"
+    )
+}
+
+def buildIntegrationDocker() {
+    def docker_gid = sh(
+        returnStdout: true,
+        script: 'getent group docker | awk -F":" "{print \\$3}"'
+    ).trim()
+    def dockerfile_src = 'devops/ci/Dockerfile.integration.tmpl'
+    def dockerfile = 'devops/ci/Dockerfile.integration'
+    sh("sed 's~{parent}~${dockerImageUnitName}~g' ${dockerfile_src}  > ${dockerfile}")
+    return buildDocker(
+        dockerImageIntegrationName,
+        dockerfile,
+        "devops/jenkins/cortx_jenkins/agent",
+        " --build-arg docker_gid=$docker_gid"
     )
 }
 
 def cleanDocker(){
-    sh("docker rmi -f ${dockerImage} || true")
+    sh("docker rmi -f ${dockerImageUnitName} || true")
+    sh("docker rmi -f ${dockerImageIntegrationName} || true")
```

Comment:
Let's push the images to GitHub registry.
@shailesh-vaidya could you help with registry details?